tomcat/src/main/java/org/apache/coyote/http11/response/ResponseStatus.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.coyote.http11.response;
+
+public enum ResponseStatus {
+
+    OK(200, "OK"),
+    FOUND(302, "FOUND"),
+    UNAUTHORIZED(401, "UNAUTHORIZED"),
+    BAD_REQUEST(400, "BAD REQUEST"),
```

Comment:
[여기 링크](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Status) 참고하시면 정확한 HTTP 응답 상태 메시지 참고하실 수 있어요! 

```suggestion
    OK(200, "OK"),
    FOUND(302, "Found"),
    UNAUTHORIZED(401, "Unauthorized"),
    BAD_REQUEST(400, "Bad Request"),
```

Reply:
저는 OK가 대문자라서.. 다들 대문자로 당연히 쓰는 줄 알았네요... ㅋㅋㅋㅋㅋㅋㅋ..
찾아보니 OK 자체가 대문자로 명사더라고요.. 수정했습니다!!!...

커밋: [67733bc](https://github.com/woowacourse/java-http/pull/875/commits/67733bc05649dc2bc677b8f37b1a5eb4cd2b9613)